.github/workflows/release.yml

```diff
@@ -0,0 +1,84 @@
+name: Release
+
+on: 
+  push: 
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install rotp
+      run: gem install rotp
+    - name: Configure gem credentials
+      run: |
+        if [ -f ~/.gem/credentials ]; then
+          echo 'Using credentials in ~/.gem/credentials'
+        else
+          echo -e "---\n:rubygems_api_key: ${{ secrets.RUBYGEMS_API_KEY }} > ~/.gem/credentials
```

Comment:
This line seems to be missing the second " for the echo
Should be `echo -e "---\n:rubygems_api_key: ${{ secrets.RUBYGEMS_API_KEY }" > ~/.gem/credentials` I believe

Reply:
Fixed